src/doc/src/reference/environment-variables.md

```diff
@@ -342,6 +342,7 @@ let out_dir = env::var("OUT_DIR").unwrap();
   values built-in to the compiler (which can be seen with `rustc --print=cfg`)
   and values set by build scripts and extra flags passed to `rustc` (such as
   those defined in `RUSTFLAGS`). Some examples of what these variables are:
+    * `CARGO_CFG_FEATURE` --- Each activated feature of the package being built.
```

Comment:
> Some cfg values like debug_assertions, test, and Cargo features like feature="foo" are not available.

We did this a while back in <https://github.com/rust-lang/cargo/pull/14676>. And Cfg `test` and `debug_assertions` are not there anyway, so `CARGO_CFG_*` still doesn't cover everything. 

May need to remove the mention of features from that part of the doc as well.

Reply:
The fun of reading stable docs.

Ah, it was also at the end so I didn't notice it when reading the source.